man/logind.conf.xml

```diff
@@ -297,6 +297,14 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>SessionsMax=</varname></term>
+
+        <listitem><para>Controls the maximum number of concurrent user sessions to manage. Defaults to 8192
+        (8K). Depending on PAM configuration further session logins will either be refused, or permitted but not
+        tracked by <filename>systemd-logind</filename>.</para></listitem>
```

Comment:
I think the last sentence is likely to be misunderstood. Maybe something like this:
"Depending on how the <filename>pam_systemd.so</filename> module is included in the PAM stack configuration, further login sessions  will either be refused, or permitted but not tracked by <filename>systemd-logind</filename>."